### ISSUE TITLE
Execution preview is not loaded when system run in iframe

### ISSUE THREAD

**Issue by ponix-net** [n8n-io/n8n#5584]
**Describe the bug**
When running system in iframe of another web page the execution preview is not loaded in executions view.

**To Reproduce**

1. Run n8n server.
2. Create web page with iframe with source pointing to executions view (like 'http://localhost:5678/workflow/1/executions/23').
3. Loader is displayed.

**Expected behavior**
Execution preview is displayed.

**Environment (please complete the following information):**

- OS: Ubuntu Linux 22.04
- n8n Version 0.217.2
- Node.js Version 18.14.2
- Operation mode own

**Probable cause and proposed solution**
Execution preview is run also in iframe and sends message to *top* window to say that it was loaded (*n8nReady*). It should send message to *parent* instead of *top* window.
I've changed lines 3871-3872 in file NodeView.vue to:
```
3871:  if (window.parent) {
3872:      window.parent.postMessage(
```
and it worked as expected, execution preview was loaded.

**Comment by Joffcom** [comment]
Hey @ponix-net,

Thanks for reporting this one, I have created an internal ticket for this which we will be tracking as `N8N-6216`.

**Comment by janober** [comment]
Fix got released with [`n8n@0.223.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.223.0)

**Comment by Joffcom** [comment]
Hey @ponix-net,

This should now be resolved, I am going to mark this one as closed. If you still this issue let me know and we can open the issue again.
